CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Fee sniping discouraging through nLockTime - if the user specifies a `current_height`, we use that as a nlocktime, otherwise we use the last sync height (or 0 if we never synced)
 - Fix hang when `ElectrumBlockchainConfig::stop_gap` is zero.
 - Set coin type in BIP44, BIP49, and BIP84 templates
+- Return balance in separate categories, namely `available`, `trusted_pending`, `untrusted_pending` & `immature`.  
```

Comment:
`available` has been renamed to `confirmed`

Also I can see there are some whitespaces left at the end of the line, if you could remove them as well it would be great!